From 1fedf9221d25b92a2d259389f9a093b228f1fc53 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sat, 23 Dec 2017 23:04:32 +0200
Subject: [PATCH 144/249] lineage: Remove external/lzo

* This was used by external/openvpn
* We do not ship that since cm-11.0, so kill it

Change-Id: Id24e97b91e953b61e12023cbd8d0bb3a5031e6a0
---
 snippets/lineage.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index 376ad02..267f631 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -14,7 +14,6 @@
   <project path="external/libnfc-nxp" name="LineageOS/android_external_libnfc-nxp" />
   <project path="external/libtar" name="LineageOS/android_external_libtar" />
   <project path="external/lineage-sdk-api-coverage" name="LineageOS/android_external_lineage-sdk-api-coverage" />
-  <!--<project path="external/lzo" name="LineageOS/android_external_lzo" />-->
   <project path="external/nano" name="LineageOS/android_external_nano" />
   <project path="external/ntfs-3g" name="LineageOS/android_external_ntfs-3g" />
   <!--<project path="external/p7zip" name="LineageOS/android_external_p7zip" />-->
-- 
2.17.1

